.gitignore

```diff
@@ -24,5 +24,6 @@ private.properties
 .gradle/
 .idea/libraries/
 main/build/
+main/proguard/
```

Comment:
I used the process described in the README.md. I build my debug APK with this:
Apache Ant(TM) version 1.9.4 compiled on April 29 2014